js/wp-seo-post-scraper-305.js

```diff
@@ -186,13 +186,17 @@
 	/**
 	 * Gets content from the content field, if tinyMCE is initialized, use the getContent function to get the data from tinyMCE
 	 * If tiny is hidden, take the value from the contentfield, since tinyMCE isn't updated when it isn't visible.
+	 * TinyMCE doesn't update the content field realtime, it has a delay in updating this field, so getting the value of the content
+	 * field isn't always reliable.
 	 * @returns {String}
 	 */
 	PostScraper.prototype.getContentTinyMCE = function() {
 		var val = document.getElementById( 'content' ) && document.getElementById( 'content' ).value || '';
 		if ( typeof tinyMCE !== 'undefined' && typeof tinyMCE.editors !== 'undefined' && tinyMCE.editors.length !== 0) {
 			var tinyMceContent = tinyMCE.get( 'content' );
-			val = tinyMceContent && tinyMceContent.hidden === false && tinyMceContent.getContent() || '';
+			if( tinyMceContent !== null && !tinyMceContent.isHidden() ) {
```

Comment:
Since the functions returns a boolean we check it always in this way. Both will work, so for now I will keep it like this.